.travis.yml

```diff
@@ -32,6 +32,12 @@ jobs:
         - PYTHON_VERSION=3.8
         - RUN_TESTS=true
 
+      - os: linux
```

Comment:
Seems that the indentation here is two-spaces ahead.  This is confusing travis when that yml file is parsed. 

```suggestion
    - os: linux
```

Reply:
Committing this change triggered the Travis tests.